sweepai/utils/openai_proxy.py

```diff
@@ -0,0 +1,19 @@
+import openai
+
+class OpenAIProxy:
+    def __init__(self, azure_keys, fallback):
```

Comment:
the interface for this should be changed to:

1. use these as class attributes:
OPENAI_API_KEY_GPT35,
    OPENAI_API_KEY_GPT4,
    OPENAI_API_KEY_GPT4_32K,
    OPENAI_FALLBACK
2. check each model and then perform the fallback logic per model. if fallback is enabled, go to openai second. otherwise make no changes and directly call openai